app/models/service_template_transformation_plan_task.rb

```diff
@@ -34,9 +34,75 @@ def task_active
     vm_resource.update_attributes(:status => ServiceResource::STATUS_ACTIVE)
   end
 
+  def conversion_host
+    Host.find_by(:id => options[:transformation_host_id])
+  end
+
+  def transformation_log
+    host = conversion_host
+    if host.nil?
+      msg = "Conversion host was not found: ID [#{options[:transformation_host_id]}]. Download of transformation log aborted."
+      _log.error(msg)
+      raise MiqException::Error, msg
+    end
+
+    userid, password = host.auth_user_pwd(:remote)
+    if userid.blank? || password.blank?
+      msg = "Host credential is required: userid [#{userid}], password [#{password}]. Download of transformation log aborted."
+      _log.error(msg)
+      raise MiqException::Error, msg
+    end
+
+    logfile = options.fetch_path(:virtv2v_wrapper, "v2v_log")
+    if logfile.blank?
+      msg = "The location of transformation log was not set. Download of transformation log aborted."
+      _log.error(msg)
+      raise MiqException::Error, msg
+    end
+
+    begin
+      require 'net/scp'
+      Net::SCP::download!(host.ipaddress, userid, logfile, nil, :ssh => {:password => password})
+    rescue => scp_err
```

Comment:
Per Rubocop I think we should be rescuing `Net::SCP::Error` error types here.

Also, is the `Net::SCP::download!` form of the call the only one that downloads into a local buffer or does `Net::SCP.download!` do it as well?

Reference: https://github.com/net-ssh/net-scp#synopsis